lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-hawkular-middleware_manager.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Hawkular_MiddlewareManager <
+      MiqAeServiceModelBase
```

Comment:
this should just be on the previous line...we don't have 80 char hard limit...closer to 120 char soft limit